pages/features.html

```diff
@@ -113,86 +142,53 @@
     <div style="clear: both;"></div>
 
     <div class="gapfiller">
-      <H1 style="margin-left: 30px;">{% trans "Designer Skins" %}</H1>
+      <h1 style="margin-left: 30px;">{% trans "Designer Skins" %}</h1>
       <center>
         <img width="800px" src="{% static '/static/images/skins.png' %}">
-        <p>{% trans "Includes 3 exclusive designer skins with configurable layouts" %}.</p>
+        <p>{% trans "Three exclusive designer skins with configurable layouts" %}</p>
       </center>
     </div>
 
     <!-- YouTube video
+    TODO: New video for 2.0
     <div style="padding-bottom: 30px;">
       <center>
         <iframe width="640" height="360" src="http://www.youtube.com/embed/axlW5Q_8y6c?start=499&hd=1&modestbranding=0&rel=0&controls=1" frameborder="0" allowfullscreen></iframe>
       </center>
     </div> -->
 
-    <H1>{% trans "Record and Broadcast your Mix" %}</H1>
+    <h1>{% trans "Record and Broadcast your Mix" %}</h1>
     <p>
-      {% trans "Mixxx includes a number of recording and broadcasting features because you can't get your name out there without sharing your DJ mixes." %}
+      {% trans "Mixxx includes a number of recording and broadcasting features to help you share your mixes and spread your name." %}
     </p>
+    
     <div style="clear: both;"></div>
 
     <div class="halfbox_left">
-      <H2><img src="{% static '/static/images/ic_library_recordings.png' %}" class="feature_icon">Recording</H2>
-      <p>{% trans "Record your live mixes to the lossless WAVE or compressed Ogg Vorbis formats, and optional MP3 support. Recordings are automatically archived in the library for fast access." %}
+      <h2><img src="{% static '/static/images/ic_library_recordings.png' %}" class="feature_icon">Recording</h2>
+      <!--TODO: clarify. What does "optional" mean?-->
```

Comment:
How about:
"Record your live mixes to the lossless WAVE or compressed Ogg Vorbis formats, with MP3 recording enabled by separate (external?) LAME library"